unittest/gunit/table_cache-t.cc

```diff
@@ -819,19 +819,11 @@ TEST_F(TableCacheDoubleCacheDeathTest, ManagerFreeTable)
   // There should be assert failure since we are trying
   // to free all tables for share_1, while some tables
   // are in use.
-<<<<<<< HEAD
-#ifndef DBUG_OFF
-  MY_EXPECT_DEATH_IF_SUPPORTED(table_cache_manager.free_table(thd_1,
-||||||| e5d189ecb94
-#ifndef DBUG_OFF
-  EXPECT_DEATH_IF_SUPPORTED(table_cache_manager.free_table(thd_1,
-=======
 #ifndef NDEBUG
-  EXPECT_DEATH_IF_SUPPORTED(table_cache_manager.free_table(thd_1,
->>>>>>> 37b047220a907c2a6d7235ddf2b7a6be916cc82e
-                                                           TDC_RT_REMOVE_ALL,
-                                                           &share_1),
-                            ".*Assertion.*is_empty.*");
+  MY_EXPECT_DEATH_IF_SUPPORTED(table_cache_manager.free_table(thd_1,
```

Comment:
Likewise

Reply:
Please, I have included the result of this test in the output above.